content/en/users/aai/check-in/vos/_index.md

```diff
@@ -106,6 +96,36 @@ The VO enrollment URL can be found through the EGI Check-in Registry:
 
    ![Member role active](./check-in-role-active.png)
 
+### Reviewing VO membership requests
```

Comment:
Why can't you see all the sub-paragraphs in the menu on the left? They need to be made visible, please.

Reply:
The page-specific sections are shown in the right menu, is this what you meant?
![Screenshot 2022-04-06 at 16 37 20](https://user-images.githubusercontent.com/451417/162000185-a86e801c-2ea1-4d4f-9b7f-bc73b4468a3a.png)

Reply:
Sorry @gwarf , it's in "Joining a VO" that I don't see the subparagraphs.
The following two sub-paragraphs are required:

- Joining a VO in COmanage (and not in Check-in!)
- Joining to VO in PERUN

Reply:
Yes, I will create later a new PR for PERUN documentation.

Reply:
There is already an existing PR for PERUN, please review it: https://github.com/EGI-Federation/documentation/pull/398